src/ConnectedField.types.js.flow

```diff
@@ -18,11 +18,11 @@ export type Props = {
   // same as Props in createFieldProps.js:
   asyncError: any,
   asyncValidating: boolean,
-  onBlur: { (event: Event, newValue: ?any, previousValue: ?any): void },
-  onChange: { (event: Event, newValue: ?any, previousValue: ?any): void },
-  onDrop: { (event: Event, newValue: ?any, previousValue: ?any): void },
-  onDragStart: { (event: Event): void },
-  onFocus: { (event: Event): void },
+  onBlur: { (event: Event, newValue: ?any, previousValue: ?any, name: string): void },
+  onChange: { (event: Event, newValue: ?any, previousValue: ?any, name: string): void },
+  onDrop: { (event: Event, newValue: ?any, previousValue: ?any, name: string): void },
```

Comment:
Does this work well in Flow? A required argument after an optional one 🤔 

I have zero experience with Flow, only TypeScript in typed-JS world and it definitely wouldn't work 😛